purchase_order_line_image/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Ooops - Ashish Hirpara <ashish.hirapara1995@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Order Line Images",
+    "summary": """Order Line Product Images In Purchase Order.""",
+    "author": "ooops404, Ashish Hirapara, Odoo Community Association (OCA), Cetmix",
+    "version": "14.0.1.0.0",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "category": "Purchase",
+    "license": "AGPL-3",
+    "depends": ["purchase", "purchase_discount"],
```

Comment:
Why depending on purchase_discount?

Reply:
Thanks for the comment, I removed this dependency